pkg/crc/ssh/ssh.go

```diff
@@ -20,9 +22,29 @@ func CreateRunnerWithPrivateKey(driver drivers.Driver, privateKey string) *SSHRu
 
 // Create a host using the driver's config
 func (runner *SSHRunner) Run(command string) (string, error) {
-	return drivers.RunSSHCommandFromDriver(runner.driver, runner.privateSSHKey, command)
+	return runSSHCommandFromDriver(runner.driver, runner.privateSSHKey, command)
 }
 
 func (runner *SSHRunner) SetPrivateKeyPath(path string) {
 	runner.privateSSHKey = path
 }
+
+func runSSHCommandFromDriver(d drivers.Driver, sshPrivateKeyPath, command string) (string, error) {
+	client, err := drivers.GetSSHClientFromDriver(d, sshPrivateKeyPath)
+	if err != nil {
+		return "", err
+	}
+
+	logging.Debugf("About to run SSH command:\n%s", command)
+
+	output, err := client.Output(command)
+	logging.Debugf("SSH cmd err, output: %v: %s", err, output)
```

Comment:
imo that's a good opportunity to cleanup a bit the debug output which is far too verbose.

Reply:
@cfergeau anything particular in mind since it show what command is going to run and then tell about error/output of that command, I think that much info we do need right?

Reply:
Ah the logs I have in mind might come from libmachine generic rpc code, I'd have to look again at a debug log to remember what I find too verbose :)